src/Stack/Config.hs

```diff
@@ -75,7 +75,7 @@ import           Network.HTTP.Client (parseUrlThrow)
 import           Network.HTTP.Download (download)
 import           Network.HTTP.Simple (httpJSON, getResponseBody)
 import           Options.Applicative (Parser, strOption, long, help)
-import           Path
+import           Path hiding (fileExtension)
```

Comment:
Looks like we're getting some warnings on [Travis](https://travis-ci.org/commercialhaskell/stack/jobs/189485298):

```
src/Stack/Config.hs:78:1: Warning:
    Module ‘Path’ does not export ‘fileExtension’
```